code/game/objects/items/stacks/stack.dm

```diff
@@ -339,7 +339,7 @@
 	return ..()
 
 /obj/item/stack/restock()
-	if(istype(src,/obj/item/stack/cable_coil) || istype(src,/obj/item/stack/medical))
+	if(istype(src,/obj/item/stack/cable_coil) || istype(src,/obj/item/stack/medical) || istype(src,/obj/item/stack/package_wrap))
```

Comment:
If you don't typecheck it will restock any and all stacks. Doesn't even need to be a module.